lib/sqlalchemy/sql/_selectable_constructors.py

```diff
@@ -443,19 +442,23 @@ def select(
     __ent8: _TCCA[_T8],
     __ent9: _TCCA[_T9],
     /,
-) -> Select[Tuple[_T0, _T1, _T2, _T3, _T4, _T5, _T6, _T7, _T8, _T9]]:
+) -> Select[_T0, _T1, _T2, _T3, _T4, _T5, _T6, _T7, _T8, _T9]:
     ...
 
 
 # END OVERLOADED FUNCTIONS select
 
 
 @overload
-def select(*entities: _ColumnsClauseArgument[Any], **__kw: Any) -> Select[Any]:
+def select(
+    *entities: _ColumnsClauseArgument[Any], **__kw: Any
```

Comment:
I think this **_kw workaround is no longer needed since https://github.com/sqlalchemy/sqlalchemy/commit/416c5787b6177d9503b26aa3da4e9284537eaa0a